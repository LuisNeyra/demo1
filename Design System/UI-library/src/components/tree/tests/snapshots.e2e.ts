import { snapIt } from '../../../test';

const component = 'tree';

describe(component, () => {
  const snap = snapIt(component);

  const html = `
<blaze-tree>
  <blaze-tree-leaf>File 1</blaze-tree-leaf>
  <blaze-tree-leaf>File 2</blaze-tree-leaf>
  <blaze-tree-leaf>File 3</blaze-tree-leaf>
  <blaze-tree-branch>
    <span slot="branch">Directory 1</span>
    <blaze-tree-leaf slot="leaf">File 4</blaze-tree-leaf>
    <blaze-tree-leaf slot="leaf">File 5</blaze-tree-leaf>
    <blaze-tree-leaf slot="leaf">File 6</blaze-tree-leaf>
  </blaze-tree-branch>
  <blaze-tree-branch>
    <span slot="branch">Directory 2</span>
    <blaze-tree-leaf slot="leaf">File 7</blaze-tree-leaf>
    <blaze-tree-leaf slot="leaf">File 8</blaze-tree-leaf>
    <blaze-tree-leaf slot="leaf">File 9</blaze-tree-leaf>
    <blaze-tree-branch slot="leaf">
      <span slot="branch">Sub-directory 1</span>
      <blaze-tree-leaf slot="leaf">File 10</blaze-tree-leaf>
      <blaze-tree-leaf slot="leaf">File 11</blaze-tree-leaf>
      <blaze-tree-leaf slot="leaf">File 12</blaze-tree-leaf>
    </blaze-tree-branch>
  </blaze-tree-branch>
</blaze-tree>
`;

  describe('renders', () => {
    snap('tree', html);
  });
});
