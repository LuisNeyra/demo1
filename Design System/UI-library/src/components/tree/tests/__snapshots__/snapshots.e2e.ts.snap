// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tree renders tree 1`] = `"<blaze-tree class=\\"hydrated\\"><!----><div role=\\"tree\\" class=\\"c-tree\\"> <blaze-tree-leaf class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 1</span></blaze-tree-leaf> <blaze-tree-leaf class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 2</span></blaze-tree-leaf> <blaze-tree-leaf class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 3</span></blaze-tree-leaf> <blaze-tree-branch class=\\"hydrated\\"><!----> <button role=\\"treeitem\\" aria-expanded=\\"false\\" class=\\"c-tree__item\\"><span slot=\\"branch\\">Directory 1</span></button><div role=\\"tree\\" class=\\"c-tree\\"><blaze-tree-leaf slot=\\"leaf\\" class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 4</span></blaze-tree-leaf><blaze-tree-leaf slot=\\"leaf\\" class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 5</span></blaze-tree-leaf><blaze-tree-leaf slot=\\"leaf\\" class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 6</span></blaze-tree-leaf></div></blaze-tree-branch> <blaze-tree-branch class=\\"hydrated\\"><!----> <button role=\\"treeitem\\" aria-expanded=\\"false\\" class=\\"c-tree__item\\"><span slot=\\"branch\\">Directory 2</span></button><div role=\\"tree\\" class=\\"c-tree\\"><blaze-tree-leaf slot=\\"leaf\\" class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 7</span></blaze-tree-leaf><blaze-tree-leaf slot=\\"leaf\\" class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 8</span></blaze-tree-leaf><blaze-tree-leaf slot=\\"leaf\\" class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 9</span></blaze-tree-leaf><blaze-tree-branch slot=\\"leaf\\" class=\\"hydrated\\"><!----> <button role=\\"treeitem\\" aria-expanded=\\"false\\" class=\\"c-tree__item\\"><span slot=\\"branch\\">Sub-directory 1</span></button><div role=\\"tree\\" class=\\"c-tree\\"><blaze-tree-leaf slot=\\"leaf\\" class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 10</span></blaze-tree-leaf><blaze-tree-leaf slot=\\"leaf\\" class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 11</span></blaze-tree-leaf><blaze-tree-leaf slot=\\"leaf\\" class=\\"hydrated\\"><!----><span role=\\"treeitem\\" class=\\"c-tree__item\\">File 12</span></blaze-tree-leaf></div></blaze-tree-branch></div></blaze-tree-branch> </div></blaze-tree>"`;
